workos/utils/http_client.py

```diff
@@ -168,7 +174,7 @@ async def request(
         params: ParamsType = None,
         json: JsonType = None,
         headers: HeadersType = None,
-        token: Optional[str] = None,
+        exclude_default_auth_headers: bool = False,
```

Comment:
New param used for UM auth methods/a few other methods where we don't need the regular Auth header.

Reply:
Ah, yeah. UM complicates this.